<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeEquipmentDialog</class>
 <widget class="QWidget" name="ChangeEquipmentDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Equipment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QComboBox" name="combo_actor"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Actor:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Equipment</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radio_weapon">
          <property name="text">
           <string>Weapon</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radio_shield">
          <property name="text">
           <string>Shield</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radio_helmet">
          <property name="text">
           <string>Helmet</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="radio_body_armor">
          <property name="text">
           <string>Body Armor</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="radio_accessory">
          <property name="text">
           <string>Accessory</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="combo_weapon">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="combo_shield">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="combo_helmet">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="combo_body_armor">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="combo_accessory">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_actor</tabstop>
  <tabstop>radio_weapon</tabstop>
  <tabstop>combo_weapon</tabstop>
  <tabstop>radio_shield</tabstop>
  <tabstop>combo_shield</tabstop>
  <tabstop>radio_helmet</tabstop>
  <tabstop>combo_helmet</tabstop>
  <tabstop>radio_body_armor</tabstop>
  <tabstop>combo_body_armor</tabstop>
  <tabstop>radio_accessory</tabstop>
  <tabstop>combo_accessory</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>button_ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radio_weapon</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_weapon</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_shield</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_shield</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_helmet</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_helmet</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_body_armor</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_body_armor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_accessory</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_accessory</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
